src/dmd/module.h

```diff
@@ -100,7 +100,7 @@ class Module : public Package
     Strings *versionidsNot;     // forward referenced version identifiers
 
     Macro *macrotable;          // document comment macros
-    Escape *escapetable;        // document comment escapes
+    Escape escapetable;        // document comment escapes
```

Comment:
Classes are values in C++, this should still be`*` no?

Reply:
`Escape` is a struct in D.